packages/styles/components/Alert.module.scss

```diff
@@ -0,0 +1,57 @@
+@import '~@venice/styles/core';
+
+$colors: 'primary' var(--primary-color), 'secondary' var(--secondary-color),
+  'success' var(--success-color), 'danger' var(--danger-color);
+
+$iconSize: 16px;
+$iconSizeClose: #{$iconSize * 0.75};
+
+.alert {
+  background: var(--gray-1);
+  border: 1px solid var(--gray-2);
+  border-radius: var(--border-radius);
+  font: var(--typography-body-font);
+  padding: var(--space-sm);
+  position: relative;
+
+  &.with-icon {
+    padding-left: calc(#{$iconSize} + var(--space-md));
+  }
+
+  &.with-close {
+    padding-right: calc(#{$iconSize} + var(--space-md));
+  }
+
+  @each $type, $color in $colors {
+    &.#{$type} {
+      .icon-alert {
+        color: $color;
+      }
+    }
+  }
+}
+
+.icon {
+  display: grid;
```

Comment:
why we are using `display: grid` here?

Reply:
Without Grid
<img width="207" alt="Screen Shot 2020-08-12 at 12 09 15" src="https://user-images.githubusercontent.com/2536827/90032511-d41a8600-dc94-11ea-910c-393227e29ed6.png">

With Grid
<img width="304" alt="Screen Shot 2020-08-12 at 12 09 22" src="https://user-images.githubusercontent.com/2536827/90032517-d54bb300-dc94-11ea-9053-9e2196d26793.png">

Aproach with `text-align:center; line-height:16px`
<img width="316" alt="Screen Shot 2020-08-12 at 12 10 10" src="https://user-images.githubusercontent.com/2536827/90032520-d5e44980-dc94-11ea-8709-538d0fc3b77a.png">

Reply:
uhmmmm ok, but this behavior is equal in other browsers like Firefox or Safari?

Reply:
I change aproach, still working in Chome, Firefox, Safari and edge